src/main/java/core/basesyntax/bookstore/model/Book.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Table(name = "books")

```

Comment:
https://medium.com/@centizennationwide/mysql-naming-conventions-e3a6f6219efe use singular work for table discription